tests/typecheck/test_test.yml

```
@@ -25,3 +25,44 @@
             reveal_type(self.client) # N: Revealed type is "django.test.client.Client"
             response = self.client.get('/', format="json")
             self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+- case: test_requestfactory_http_verbs
+  parametrized:
+  - method: get
+  - method: post
+  - method: put
+  - method: patch
+  - method: delete
+  - method: options
+  main: |
+    from django.urls import reverse_lazy
+    from rest_framework import test, status
+
+    url = reverse_lazy("api:example")
+    request_factory = test.APIRequestFactory()
+    request_factory.{{ method }}(url)
```

Optional nit: the test runner is quite slow to set up each test case, it's better to merge this a s a single test, e.g.

```suggestion
    request_factory.get(url)
    request_factory.post(url)
    request_factory.put(url)
    ...
```